spec/lib/metasploit/framework/login_scanner/jenkins_spec.rb

```diff
@@ -2,9 +2,103 @@
 require 'metasploit/framework/login_scanner/jenkins'
 
 RSpec.describe Metasploit::Framework::LoginScanner::Jenkins do
+  subject(:http_scanner) { described_class.new('host' => 'nosuchplace.metasploit.com') }
```

Comment:
if you want a url that can't be resolved we should probably use a `.example.com` or similar reserved domains

Reply:
why? this is literally used in every other test

Reply:
I don't mind much either way, I didn't realise you'd picked it up from elsewhere in framework, just pointing out that there are reserved domains specifically for that purpose is all

A quick search of the spec folder shows `nosuchplace.metasploit.com` being used twice and `example.com` being used much more, but like I said we've already got it in there so I don't mind